client/blocks/plugins-scheduled-updates-multisite/schedule-list.tsx

```diff
@@ -88,16 +89,17 @@ export const ScheduleList = ( props: Props ) => {
 
 	return (
 		<div className="plugins-update-manager plugins-update-manager-multisite">
-			<h1 className="wp-brand-font">List schedules</h1>
-			<Button
-				__next40pxDefaultSize
-				icon={ plus }
-				variant="primary"
-				onClick={ onCreateNewSchedule }
-				disabled={ false }
-			>
-				{ translate( 'Add new schedule' ) }
-			</Button>
+			<div className="plugins-update-manager-multisite-header">
```

Comment:
Should we use BEM notation here? `plugin-update-manager-multisite__header`? Just a question, it always confuses me when to do this and when not do.

Reply:
As discussed in p1714548529945449-slack-C01A60HCGUA we'll fix naming issues when everything is done and in the polish phase.